### ISSUE TITLE
Ajuda

### ISSUE THREAD

**Issue by bfguimaraes** [rodrigordi/opennfe#5]
Alguém teria tempo para me ajudar a entender um pouco do trabalho? Eu estou tento um dificuldade para executar o código, estou muito interessado no projeto e acredito que posso ajudar no desenvolvimento.

Se alguém puder, meu Skype é o bferreiraguimaraes

**Comment by rodrigordi** [comment]
@bfguimaraes, o projeto pode ser executado como um executavel ligado ao um banco de dados (mssql ou mysql) ou pode ser utilizado dentro da sua aplicação utilizando a classe de interface. Como você está utilizando ?

**Comment by bfguimaraes** [comment]
@rodrigordi Eu entendi, mas eu não entendi muito bem como se faz isso, não ta tão especificado assim no READ ME.

**Comment by rodrigordi** [comment]
@bfguimaraes verdade. O ReadME está falando somente sobre o uso dos compilados. Vamos atualizar isso então. Mas basicamente, eu peguei o projeto RDI.NFe2.Interface.ComObject e adicionei ao meu projeto ERP.  E depois para usar as funções eu instancio um Utilitario e depois chamo as funções de envio, consulta, etc..

var proxy = new RDI.NFe2.Business.Utilitario();
                var stRawCertificado = oEmpresa.oCertificado.raw;
                var senhaCertificado = oEmpresa.oCertificado.password;
                TCodUfIBGE uf = (TCodUfIBGE)oEmpresa.oEstado.codigoIBGE;
                proxy.SetUtilitario_v5(stRawCertificado + "|" + senhaCertificado, false, producao, modoEmissao, uf.ToString(), TipoConexao.NFe, OpenSigner.BuscaCertificado.Raw, 3);

 
![image](https://user-images.githubusercontent.com/1608626/30985341-66732cec-a466-11e7-9baa-3304861a54f1.png)

**Comment by bfguimaraes** [comment]
@rodrigordi Desculpe cara, mas ainda não consegui.

**Comment by rodrigordi** [comment]
@bfguimaraes, o que acontece ? coloca um print do problema. eu tentarei te ajudar! :D

**Comment by luizfelipedev-agillity** [comment]
Ainda Funciona esse sistema??? não recebe atualizações a algum tempo

**Comment by mrocha-rdi** [comment]
@luizfelipedev-agillity atualizamos para a Reforma Tributária. Tudo está sendo disponibilizado primeiro no site https://opennfe.com

Mas já já aqui também estará atualizado. Segue a gente nas redes sociais que as notícias vão chegar mais rápido.
